### DISCUSSION TITLE
最佳测速时间？

### DISCUSSION THREAD

**Discussion by 1c383r9** [XIU2/CloudflareSpeedTest#125] [提问]
如果一天测一次速，什么时间段测速比较合理？

晚高峰测速得到的ip科学吗？我目前是在21点测速。

**Comment by XIU2** [comment]
严格来说没有最佳测速时间。
这主要取决于你平时上网的时间段。
如果你一般是晚上玩，那么建议晚上去测速，如果是白天就是白天测速喽~
高低峰期差别还是挺大的，所以要根据你的主要上网时间来选择测速时间。
